lib/Payment.js

```diff
@@ -775,6 +775,25 @@ class Payment {
       }
     );
   }
+
+  /**
+   * wechat international merchant only
+   * retrieve exchange rate for given foreign currency and date
```

Comment:
The first letter of the first word, better to change it to Capital(e.g: Retrieve exchange...) to be consistent with other jsdoc